examples/knowledge_graph/data/kg_vertex_types.csv

```diff
@@ -1,9 +1,8 @@
 v_high_level_category,v_type,v_type_def,v_type_def_source
-program_interpretable,math_function,"A function is a relation that uniquely associates members of one set with members of another set. More formally, a function from  A to B is an object f  such that every a in A  is uniquely associated with an object f(a) in B. A function is therefore a many-to-one (or sometimes one-to-one) relation.",http://mathworld.wolfram.com/Function.html
-program_interpretable,program,A computer program is a collection of instructions that performs a specific task when executed by a computer.,https://en.wikipedia.org/wiki/Computer_program
-program_interpretable,primitve_data_type,"A primitive type is a concrete type whose data consists of plain old bits. Classic examples of primitive types are integers and floating-point values. Unlike most languages, Julia lets you declare your own primitive types, rather than providing only a fixed set of built-in ones. ",https://docs.julialang.org/en/v1/manual/types/index.html#Primitive-Types-1
-program_interpretable,data_structure,"In computer science, a data structure is a data organization, management and storage format that enables efficient access and modification. More precisely, a data structure is a collection of datavalues, the relationships among them, and the functions or operations that can be applied to the data.",https://en.wikipedia.org/wiki/Data_structure
-program_interpretable,control_flow_statement,"A control flow statement is a statement, the execution of which results in a choice being made as to which of two or more paths to follow.",https://en.wikipedia.org/wiki/Control_flow
+program_interpretable,math_function,"A function is a relationthat uniquely associates members of one set with members of another set. More formally, a function from A to B is an object f such that every a in A is uniquely associated with an object f(a) in B. A function is therefore a many-to-one (or sometimes one-to-one) relation.",http://mathworld.wolfram.com/Function.html
+program_interpretable,program,A computer program is a collection of instructions that performs a specific task when executed by a computer.,https://en.wikipedia.org/wiki/Computer_program
+program_interpretable,type,"In�computer science�and�computer programming, a type�is an attribute of�data�which tells the�compiler�or�interpreter�how the programmer intends to use the data. Most programming languages support common data types of�real,�integerand�boolean. A data type constrains the values that an�expression, such as a variable or a function, might take. This data type defines the operations that can be done on the data, the meaning of the data, and the way values of that type can be stored.�",https://en.wikipedia.org/wiki/Data_type

```

Comment:
I see funky unicode in this row, is this a nonbreaking space or something?

Reply:
hm, I noticed this earlier and thought was fixed. will investigate.